src/Collections/TerminusCollection.php

```diff
@@ -141,6 +141,16 @@ public function get($id)
         );
     }
 
+    /**
+     * Returns the name of the model class this collection collects
+     *
+     * @return string
+     */
+    public function getCollectedClass()
+    {
+        return $this->collected_class;
```

Comment:
I kinda think I should've made this property a const in the beginning.

Reply:
I think I probably would have made it a method, and have each subclass override the method and return a constant rather than have the subclass change the method field. Any of these options are fine, though, and are more a matter of style than anything else.

You should, however, be consistent. If you offer the getCollectedClass() method, then the code should use it everywhere. At the moment, the code in TerminusCollection bypasses the method and references the property directly, leaving the potential for inconsistency (e.g. if a subclass overrides getCollectedClass but does not change the field). You should either remove this method, or change all references to the field to call the class.

Reply:
Since you're already using the const for PRETTY_NAME, perhaps it would be worthwhile to do the same thing for collected class, for consistency.